src/context/ToolboxContext/index.tsx

```diff
@@ -47,10 +52,19 @@ const ToolboxProvider: React.FC<ToolboxProviderProps> = ({ children }) => {
   useEffect(() => {
     const fetchItems = async () => {
       if (state.selectedCollection) {
+        const extent = map.getView().calculateExtent(map.getSize());
```

Comment:
This part used to run when the user panned around the map, that functionality is no longer here, maybe it's not worth re-adding until the draw tools ticket.

Reply:
Yeah, this is a hard one, I'd like to add the draw tools very soon really. I don't think anyone really likes the calculate extent on pan/zoom. I guess it's more for the likes of Alasdair to decide on that priority. I'll look to add that back in now though.

Reply:
I've updated the code to now have a drawing tool to draw bbox AOI. This is what we ultimately want anyway, so don't see the point continuing with something we are just going to throw away soon-ish.